docs/Developers/Testing_Bears.rst

```diff
@@ -24,8 +24,9 @@ Let us understand how to write tests for ``TooManyLinesBear`` in ``some_dir``.
 
     class TooManyLinesBear(LocalBear):
 
-        def run(file,
+        def run(self,
                 filename,
+                file,
```

Comment:
Any particular reason to place this argument here ?
IMO the order should be alphabetical unless otherwise needed.
i.e
```python
def run(self,
        file,
        filename,
    ...
```

Reply:
@virresh LocalBear run method has arguments in same order. As TooManyLinesBear inherits from that, arguments should be in same order. To keep these arguments in alphabetical order I have to make a change in LocalBear class.

Reply:
Oh
I missed that 
Changes look good then :+1: 
Just the nit-pick of closes to Closes and it looks good to go then.